### PR TITLE
DOC-6862 ClusterHelper.Close()

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

**Review by Tim020 — APPROVED** [review]
Looks good to me! :)

File: modules/ROOT/pages/managing-connections.adoc [hunk 1]
```diff
@@ -158,7 +158,7 @@ Here are some very important things to keep in mind:
 The [.api]`ClusterHelper` is a singleton for a [.api]`Cluster` instance and a "multi-ton" for bucket instances; the buckets will be cached and reused.
 * The SDK is thread-safe, so no additional synchronization is needed when interacting with the SDK.
 * If different clusters need to be accessed, you can do this with multiple [.api]`Cluster` instances.
-* The [.api]`Cluster` instance and the [.api]`ClusterHelper` class should be initialized when the application starts up and closed via [.api]`Dispose()` when the application shuts down.
+* The `Cluster` instance and the `ClusterHelper` class should be initialized when the application starts up, and closed via `Cluster.Dispose()` and `ClusterHelper.Close()` respectively, when the application shuts down.
```
Comment (jeffrymorris) [inline modules/ROOT/pages/managing-connections.adoc:162]: SDK3 does not have a `ClusterHelper` (its only in SDK2 atm). Besides `Cluster.Dispose()` there is `Cluster.DisposeAsync()` which must be awaited.

**Review by jeffrymorris — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/managing-connections.adoc (hunk 1 above)
Comment (RichardSmedley) [reply modules/ROOT/pages/managing-connections.adoc:162]: Thanks @jeffrymorris - this is a 2.7 doc fix

Comment (jeffrymorris) [reply modules/ROOT/pages/managing-connections.adoc:162]: @RichardSmedley my bad - need to remember to caffeinate before reviewing!

**Review by jeffrymorris — APPROVED** [review]

**Pull request merged**